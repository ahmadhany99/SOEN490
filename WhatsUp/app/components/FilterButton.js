import React from 'react';
import { Text, TouchableOpacity, StyleSheet, Platform } from 'react-native';
import { Ionicons } from '@expo/vector-icons';

function FilterButton({ image, onPress, title}) {
    return (
        <TouchableOpacity style={[styles.button, {flexDirection:'row'}]} onPress={onPress}>
            <Text>{title}</Text>
            <Ionicons name={image} size={30} color={'#32bca5'} />
        </TouchableOpacity>
    );
}

const styles = StyleSheet.create({
    button: {
        borderRadius: 20,
        justifyContent: 'center',
        alignItems: 'center',
        marginVertical: 5,
        height: '50%',
        width: '10%',
        marginTop: 13,
        marginLeft: 8,
    },
    text: {
        color: '#32bca5',
        fontSize: 13,
        fontWeight: 'bold',
        fontFamily: Platform.OS === 'ios' ? 'Helvetica Neue' : 'sans-serif',
        marginTop: '2%',
    }
});


export default FilterButton;