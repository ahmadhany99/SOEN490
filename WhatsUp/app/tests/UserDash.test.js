import "react-native";
import React from "react";
import renderer from "react-test-renderer";
import { NavigationContainer } from "@react-navigation/native";
import {
  render,
  waitFor,
  fireEvent,
  screen,
} from "@testing-library/react-native";
import { Alert } from "react-native";
import UserDashboardScreen from "../screens/UserDashboard";
import FilterButton from "../components/FilterButton";

jest.useFakeTimers();

it("renders correctly", () => {
  const tree = render(
    <NavigationContainer>
      <UserDashboardScreen />
    </NavigationContainer>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

test("Clicking Location", async () => {
  render(
    <NavigationContainer>
      <UserDashboardScreen />
    </NavigationContainer>
  );
  await waitFor(() => {
    fireEvent.press(screen.getByText("Location, QC"));
  });
});

test("Clicking Notification", async () => {
  render(
    <NavigationContainer>
      <UserDashboardScreen />
    </NavigationContainer>
  );
  const temp = screen.queryAllByText("");
  await waitFor(() => {
    fireEvent.press(temp[0]);
  });
});

test("Clicking Filter", async () => {
  render(
    <NavigationContainer>
          <UserDashboardScreen />
    </NavigationContainer>
  );
  const temp = screen.queryAllByText("");
  await waitFor(() => {
    fireEvent.press(temp[1]);
  });
});

/**test("Clicking Event", async () => {
  render(
    <NavigationContainer>
      <UserDashboardScreen />
    </NavigationContainer>
  );
  const temp = screen.queryAllByText("");
  await waitFor(() => {
    fireEvent.press(screen.getByText("event"));
  });
});**/

test("Clicking Home", async () => {
  render(
    <NavigationContainer>
      <UserDashboardScreen />
    </NavigationContainer>
  );
  const temp = screen.queryAllByText("");
  await waitFor(() => {
    fireEvent.press(temp[2]);
  });
});

test("Clicking Ticket", async () => {
  render(
    <NavigationContainer>
      <UserDashboardScreen />
    </NavigationContainer>
  );
  const temp = screen.queryAllByText("");
  await waitFor(() => {
    fireEvent.press(temp[3]);
  });
});

test("Clicking Bookmark", async () => {
  render(
    <NavigationContainer>
      <UserDashboardScreen />
    </NavigationContainer>
  );
  const temp = screen.queryAllByText("");
  await waitFor(() => {
    fireEvent.press(temp[4]);
  });
});

test("Clicking Profile", async () => {
  render(
    <NavigationContainer>
      <UserDashboardScreen />
    </NavigationContainer>
  );
  const temp = screen.queryAllByText("");
  await waitFor(() => {
    fireEvent.press(temp[5]);
  });
});