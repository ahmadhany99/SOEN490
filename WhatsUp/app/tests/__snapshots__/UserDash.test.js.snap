// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
        "paddingTop": 20,
      },
      Object {
        "backgroundColor": "#F5F5F5",
        "padding": 20,
      },
    ]
  }
>
  <View
    style={
      Object {
        "left": "1.5%",
        "marginBottom": "40%",
        "marginTop": "5%",
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#FFFFFF",
            "borderRadius": 20,
            "elevation": 4,
            "flexDirection": "row",
            "height": 33,
            "justifyContent": "center",
            "marginVertical": 5,
            "opacity": 1,
            "padding": 1,
            "shadowColor": "#100101",
            "shadowOffset": Object {
              "height": 1,
              "width": 1,
            },
            "shadowOpacity": 0.2,
            "shadowRadius": 3,
          }
        }
      >
        <Text />
        <Text
          style={
            Array [
              Object {
                "color": "#32bca5",
                "fontFamily": "Helvetica Neue",
                "fontSize": 13,
                "fontWeight": "bold",
                "marginTop": "2%",
              },
              Object {
                "marginRight": "8%",
              },
            ]
          }
        >
          Location, QC
        </Text>
      </View>
      <View
        style={
          Object {
            "position": "absolute",
            "right": "4%",
          }
        }
      >
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFFFFF",
              "borderRadius": 20,
              "elevation": 4,
              "flexDirection": "row",
              "height": 33,
              "justifyContent": "center",
              "marginVertical": 5,
              "opacity": 1,
              "padding": 1,
              "shadowColor": "#100101",
              "shadowOffset": Object {
                "height": 1,
                "width": 1,
              },
              "shadowOpacity": 0.2,
              "shadowRadius": 3,
            }
          }
        >
          <Text />
          <Text
            style={
              Array [
                Object {
                  "color": "#32bca5",
                  "fontFamily": "Helvetica Neue",
                  "fontSize": 13,
                  "fontWeight": "bold",
                  "marginTop": "2%",
                },
                Object {
                  "marginRight": 4,
                },
              ]
            }
          />
        </View>
      </View>
    </View>
    <Text
      style={
        Object {
          "color": "#969696",
          "fontSize": 12,
          "marginTop": "6%",
        }
      }
    >
      Today's November 27
    </Text>
    <Text
      style={
        Object {
          "color": "#100101",
          "fontSize": 20,
          "fontWeight": "bold",
        }
      }
    >
      Welcome, TEMP!
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderRadius": 7,
            "elevation": 2,
            "flexDirection": "row",
            "height": 36,
            "marginVertical": 12,
            "paddingHorizontal": 12,
            "paddingVertical": 7,
            "shadowColor": "black",
            "shadowOffset": Object {
              "height": 1,
              "width": 1,
            },
            "shadowOpacity": 0.2,
            "shadowRadius": 3,
            "width": "85%",
          }
        }
      >
        <Text />
        <TextInput
          placeholder="Search for..."
          style={
            Object {
              "color": "#989797",
              "fontFamily": "Helvetica Neue",
              "marginLeft": 5,
            }
          }
        />
      </View>
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "borderRadius": 20,
            "flexDirection": "row",
            "height": "50%",
            "justifyContent": "center",
            "marginLeft": 8,
            "marginTop": 13,
            "marginVertical": 5,
            "opacity": 1,
            "width": "10%",
          }
        }
      >
        <Text />
      </View>
    </View>
    <Text
      style={
        Object {
          "color": "#100101",
          "fontSize": 16,
          "fontWeight": "bold",
          "marginTop": "6%",
        }
      }
    >
      Popular Events
    </Text>
    <RCTScrollView
      style={
        Object {
          "width": "101%",
        }
      }
    >
      <View>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFFFFF",
              "borderRadius": 19,
              "borderWidth": 0,
              "elevation": 4,
              "flexDirection": "row",
              "height": 81,
              "justifyContent": "center",
              "marginTop": 16,
              "marginVertical": 5,
              "opacity": 1,
              "padding": 1,
              "shadowColor": "#100101",
              "shadowOffset": Object {
                "height": 1,
                "width": 1,
              },
              "shadowOpacity": 0.2,
              "shadowRadius": 3,
              "width": "95%",
            }
          }
        >
          <Image
            source={1}
            style={
              Object {
                "left": 0,
                "position": "absolute",
              }
            }
          />
          <View
            style={
              Object {
                "marginLeft": 30,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#100101",
                  "fontFamily": "Helvetica Neue",
                  "fontSize": 16,
                  "fontWeight": "bold",
                  "marginTop": "2%",
                }
              }
            >
              Orientation Week
            </Text>
            <Text
              style={
                Object {
                  "color": "#969696",
                  "fontSize": 12,
                }
              }
            >
              By 
              Concordia University
            </Text>
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "marginTop": 5,
                }
              }
            >
              <Text />
              <Text
                style={
                  Object {
                    "color": "#100101",
                    "fontSize": 11,
                    "marginLeft": "2%",
                    "marginTop": "2%",
                  }
                }
              >
                May 21, 2022
              </Text>
            </View>
          </View>
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "height": 20,
                "justifyContent": "center",
                "marginLeft": 8,
                "marginTop": 13,
                "marginVertical": 5,
                "opacity": 1,
                "padding": 1,
                "position": "absolute",
                "right": 20,
                "top": 5,
                "width": 16,
              }
            }
          >
            <Text />
          </View>
        </View>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFFFFF",
              "borderRadius": 19,
              "borderWidth": 0,
              "elevation": 4,
              "flexDirection": "row",
              "height": 81,
              "justifyContent": "center",
              "marginTop": 16,
              "marginVertical": 5,
              "opacity": 1,
              "padding": 1,
              "shadowColor": "#100101",
              "shadowOffset": Object {
                "height": 1,
                "width": 1,
              },
              "shadowOpacity": 0.2,
              "shadowRadius": 3,
              "width": "95%",
            }
          }
        >
          <Image
            source={1}
            style={
              Object {
                "left": 0,
                "position": "absolute",
              }
            }
          />
          <View
            style={
              Object {
                "marginLeft": 30,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#100101",
                  "fontFamily": "Helvetica Neue",
                  "fontSize": 16,
                  "fontWeight": "bold",
                  "marginTop": "2%",
                }
              }
            >
              Orientation Week
            </Text>
            <Text
              style={
                Object {
                  "color": "#969696",
                  "fontSize": 12,
                }
              }
            >
              By 
              Concordia University
            </Text>
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "marginTop": 5,
                }
              }
            >
              <Text />
              <Text
                style={
                  Object {
                    "color": "#100101",
                    "fontSize": 11,
                    "marginLeft": "2%",
                    "marginTop": "2%",
                  }
                }
              >
                May 21, 2022
              </Text>
            </View>
          </View>
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "height": 20,
                "justifyContent": "center",
                "marginLeft": 8,
                "marginTop": 13,
                "marginVertical": 5,
                "opacity": 1,
                "padding": 1,
                "position": "absolute",
                "right": 20,
                "top": 5,
                "width": 16,
              }
            }
          >
            <Text />
          </View>
        </View>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFFFFF",
              "borderRadius": 19,
              "borderWidth": 0,
              "elevation": 4,
              "flexDirection": "row",
              "height": 81,
              "justifyContent": "center",
              "marginTop": 16,
              "marginVertical": 5,
              "opacity": 1,
              "padding": 1,
              "shadowColor": "#100101",
              "shadowOffset": Object {
                "height": 1,
                "width": 1,
              },
              "shadowOpacity": 0.2,
              "shadowRadius": 3,
              "width": "95%",
            }
          }
        >
          <Image
            source={1}
            style={
              Object {
                "left": 0,
                "position": "absolute",
              }
            }
          />
          <View
            style={
              Object {
                "marginLeft": 30,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#100101",
                  "fontFamily": "Helvetica Neue",
                  "fontSize": 16,
                  "fontWeight": "bold",
                  "marginTop": "2%",
                }
              }
            >
              Orientation Week
            </Text>
            <Text
              style={
                Object {
                  "color": "#969696",
                  "fontSize": 12,
                }
              }
            >
              By 
              Concordia University
            </Text>
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "marginTop": 5,
                }
              }
            >
              <Text />
              <Text
                style={
                  Object {
                    "color": "#100101",
                    "fontSize": 11,
                    "marginLeft": "2%",
                    "marginTop": "2%",
                  }
                }
              >
                May 21, 2022
              </Text>
            </View>
          </View>
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "height": 20,
                "justifyContent": "center",
                "marginLeft": 8,
                "marginTop": 13,
                "marginVertical": 5,
                "opacity": 1,
                "padding": 1,
                "position": "absolute",
                "right": 20,
                "top": 5,
                "width": 16,
              }
            }
          >
            <Text />
          </View>
        </View>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFFFFF",
              "borderRadius": 19,
              "borderWidth": 0,
              "elevation": 4,
              "flexDirection": "row",
              "height": 81,
              "justifyContent": "center",
              "marginTop": 16,
              "marginVertical": 5,
              "opacity": 1,
              "padding": 1,
              "shadowColor": "#100101",
              "shadowOffset": Object {
                "height": 1,
                "width": 1,
              },
              "shadowOpacity": 0.2,
              "shadowRadius": 3,
              "width": "95%",
            }
          }
        >
          <Image
            source={1}
            style={
              Object {
                "left": 0,
                "position": "absolute",
              }
            }
          />
          <View
            style={
              Object {
                "marginLeft": 30,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#100101",
                  "fontFamily": "Helvetica Neue",
                  "fontSize": 16,
                  "fontWeight": "bold",
                  "marginTop": "2%",
                }
              }
            >
              Orientation Week
            </Text>
            <Text
              style={
                Object {
                  "color": "#969696",
                  "fontSize": 12,
                }
              }
            >
              By 
              Concordia University
            </Text>
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "marginTop": 5,
                }
              }
            >
              <Text />
              <Text
                style={
                  Object {
                    "color": "#100101",
                    "fontSize": 11,
                    "marginLeft": "2%",
                    "marginTop": "2%",
                  }
                }
              >
                May 21, 2022
              </Text>
            </View>
          </View>
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "height": 20,
                "justifyContent": "center",
                "marginLeft": 8,
                "marginTop": 13,
                "marginVertical": 5,
                "opacity": 1,
                "padding": 1,
                "position": "absolute",
                "right": 20,
                "top": 5,
                "width": 16,
              }
            }
          >
            <Text />
          </View>
        </View>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFFFFF",
              "borderRadius": 19,
              "borderWidth": 0,
              "elevation": 4,
              "flexDirection": "row",
              "height": 81,
              "justifyContent": "center",
              "marginTop": 16,
              "marginVertical": 5,
              "opacity": 1,
              "padding": 1,
              "shadowColor": "#100101",
              "shadowOffset": Object {
                "height": 1,
                "width": 1,
              },
              "shadowOpacity": 0.2,
              "shadowRadius": 3,
              "width": "95%",
            }
          }
        >
          <Image
            source={1}
            style={
              Object {
                "left": 0,
                "position": "absolute",
              }
            }
          />
          <View
            style={
              Object {
                "marginLeft": 30,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#100101",
                  "fontFamily": "Helvetica Neue",
                  "fontSize": 16,
                  "fontWeight": "bold",
                  "marginTop": "2%",
                }
              }
            >
              Orientation Week
            </Text>
            <Text
              style={
                Object {
                  "color": "#969696",
                  "fontSize": 12,
                }
              }
            >
              By 
              Concordia University
            </Text>
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "marginTop": 5,
                }
              }
            >
              <Text />
              <Text
                style={
                  Object {
                    "color": "#100101",
                    "fontSize": 11,
                    "marginLeft": "2%",
                    "marginTop": "2%",
                  }
                }
              >
                May 21, 2022
              </Text>
            </View>
          </View>
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "height": 20,
                "justifyContent": "center",
                "marginLeft": 8,
                "marginTop": 13,
                "marginVertical": 5,
                "opacity": 1,
                "padding": 1,
                "position": "absolute",
                "right": 20,
                "top": 5,
                "width": 16,
              }
            }
          >
            <Text />
          </View>
        </View>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFFFFF",
              "borderRadius": 19,
              "borderWidth": 0,
              "elevation": 4,
              "flexDirection": "row",
              "height": 81,
              "justifyContent": "center",
              "marginTop": 16,
              "marginVertical": 5,
              "opacity": 1,
              "padding": 1,
              "shadowColor": "#100101",
              "shadowOffset": Object {
                "height": 1,
                "width": 1,
              },
              "shadowOpacity": 0.2,
              "shadowRadius": 3,
              "width": "95%",
            }
          }
        >
          <Image
            source={1}
            style={
              Object {
                "left": 0,
                "position": "absolute",
              }
            }
          />
          <View
            style={
              Object {
                "marginLeft": 30,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#100101",
                  "fontFamily": "Helvetica Neue",
                  "fontSize": 16,
                  "fontWeight": "bold",
                  "marginTop": "2%",
                }
              }
            >
              Orientation Week
            </Text>
            <Text
              style={
                Object {
                  "color": "#969696",
                  "fontSize": 12,
                }
              }
            >
              By 
              Concordia University
            </Text>
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "marginTop": 5,
                }
              }
            >
              <Text />
              <Text
                style={
                  Object {
                    "color": "#100101",
                    "fontSize": 11,
                    "marginLeft": "2%",
                    "marginTop": "2%",
                  }
                }
              >
                May 21, 2022
              </Text>
            </View>
          </View>
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "height": 20,
                "justifyContent": "center",
                "marginLeft": 8,
                "marginTop": 13,
                "marginVertical": 5,
                "opacity": 1,
                "padding": 1,
                "position": "absolute",
                "right": 20,
                "top": 5,
                "width": 16,
              }
            }
          >
            <Text />
          </View>
        </View>
      </View>
    </RCTScrollView>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "borderColor": "#969696",
        "borderStyle": "solid",
        "borderWidth": 0.25,
        "bottom": "-1.4%",
        "display": "flex",
        "flexDirection": "row",
        "height": "7%",
        "justifyContent": "space-between",
        "marginVertical": "1.4%",
        "paddingHorizontal": "10%",
        "position": "absolute",
        "shadowColor": "black",
        "width": "113%",
      }
    }
  >
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFFFFF",
          "borderRadius": 20,
          "flexDirection": "row",
          "height": "77%",
          "justifyContent": "center",
          "marginTop": "0.8%",
          "opacity": 1,
        }
      }
    >
      <Text />
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFFFFF",
          "borderRadius": 20,
          "flexDirection": "row",
          "height": "77%",
          "justifyContent": "center",
          "marginTop": "0.8%",
          "opacity": 1,
        }
      }
    >
      <Text />
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFFFFF",
          "borderRadius": 20,
          "flexDirection": "row",
          "height": "77%",
          "justifyContent": "center",
          "marginTop": "0.8%",
          "opacity": 1,
        }
      }
    >
      <Text />
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFFFFF",
          "borderRadius": 20,
          "flexDirection": "row",
          "height": "77%",
          "justifyContent": "center",
          "marginTop": "0.8%",
          "opacity": 1,
        }
      }
    >
      <Text />
    </View>
  </View>
</View>
`;
